hearth/lib/hearth/http/error_inspector.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+require 'time'
+
+module Hearth
+  module HTTP
```

Comment:
Error inspector was moved to HTTP because it is protocol specific - it checks http response status and specific http error classes.